packages/js/test/plugins/auctionHouseModule/createListing.test.ts

```diff
@@ -96,3 +99,150 @@ test('[auctionHouseModule] create receipt-less listings but can fetch them after
     t.ok(hasNotFoundMessage, 'has ListingReceipt Not Found message');
   }
 });
+
+test('[auctionHouseModule] create a new receipt-less Auctioneer listing on an Auction House', async (t: Test) => {
+  // Given we have an NFT.
+  const mx = await metaplex();
+  const nft = await createNft(mx);
+
+  const auctioneerAuthority = Keypair.generate();
+
+  // Create a simple Auctioneer Auction House.
+  const { client } = await createAuctionHouse(mx, auctioneerAuthority);
```

Comment:
```suggestion
  // And a simple Auctioneer Auction House.
  const auctioneerAuthority = Keypair.generate();
  const { client } = await createAuctionHouse(mx, auctioneerAuthority);
```

Could you update the tests below as well so it follows this convention:

```ts
// Given (initial condition).

// And/But (optional additional initial conditions).

// When (main trigger).

// And/But (optional additional triggers).

// Then (main assertions).

// And/But (optional additional assertions).
```